package.json

```diff
@@ -97,6 +97,6 @@
   },
   "main": "qunit/qunit.js",
   "engines": {
-    "node": "4.* || 6.* || 7.*"
```

Comment:
That fields isn't for what you explicitly support, it's what the code is compatible with.